code/modules/mob/living/status_procs.dm

```diff
@@ -146,10 +146,12 @@
 	else
 		disabilities[disability] |= list(source)
 
-/mob/living/proc/remove_disability(disability, list/sources)
+/mob/living/proc/remove_disability(disability, sources)
 	if(!disabilities[disability])
 		return
 
+	sources = list(sources)
```

Comment:
`if(!islist(sources)`?

Reply:
@XDTM ?

Reply:
If the list() proc does not do that already, then yes